tinyms/serving/client/client.py

```diff
@@ -59,7 +59,7 @@ def server_started(host='127.0.0.1', port=5000):
         return False
 
 
-def list_servables():
+def list_servables(servable_name=None, servable_path=None):
```

Comment:
It's not suggested to let users to perceive the concept of `servable_name` and `servable_path` when calling `list_servables` function.

Reply:
What you said makes me a little confused

Reply:
In the design of the TinyMS serving, users from client side don't need to know the detailed message of the servable, he could only list all servables available, find the best one, and directly pass the `servable_name` to the server side, and the server will handle all the remaining stuff.

Reply:
If you want to specify the `servable_path`, I think it would be better to assign it when calling `start_server` method.